persim/persistent_entropy.py

```diff
@@ -0,0 +1,61 @@
+"""
+
+    Implementation of persistent entropy
+
+    Author: Eduardo Paluzo Hidalgo (cimagroup, University of Seville)
+    contact: epaluzo@us.es
+
+"""
+
+import numpy as np
+
+
+__all__ = ["persistent_entropy"]
```

Comment:
this string should match the function name. I'd prefer the function name to be `persistent_entropy` like the string says.